plasmapy/atomic/atomic.py

```diff
@@ -1473,7 +1486,7 @@ def _extract_charge_state(argument):
     return argument, charge_state
 
 
-def _is_neutron(argument, mass_numb=None):
+def _is_neutron(argument: Any, mass_numb: int = None) -> bool:
```

Comment:
I haven't really gotten into the typing documentation yet, but here's a question: is there a point in hinting that this `argument` can be `Any`? Is there any added value over leaving it empty, besides being explicit, I guess?

Reply:
If we used a static type checker in PlasmaPy, I doubt it would make a difference whether we had `Any` or didn't have anything.  I think the benefit would be that it makes what's going on more clear.

This particular annotation may end up being short-lived because I'm thinking about refactoring the atomic subpackage so that we don't have to deal with all of these special private functions like `_is_neutron`.  We can probably take care of all of the annoying aliases in a much cleaner way.